run_tests.sh

```diff
@@ -18,3 +18,25 @@ elif [ "$PART" == "valgrind_pytest" ]; then
 	echo "Unknown part: $PART, known are: gdb_pytest valgrind_pytest" >&2
 	exit 1
 fi
+
+
+BUILD_DIR="$(dirname "${BASH_SOURCE[0]}")/build"
+if [ ! -d "$BUILD_DIR" ]; then
+	echo "Could not find build directory $BUILD_DIR" >&2
+	exit 1
+fi
+
+PART="$1"
+shift
+
+if [ "$PART" == "cpp" ]; then
+	cd "$BUILD_DIR" && "./src/adcc/tests/adccore_tests" "$@"
```

Comment:
Are these paths still correct?

Reply:
Obviously not. The only thing that works right now is setup.py test.

Reply:
Yes, just wanted to add a note that we don't forget...